src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Machine - excavator started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Machine - excavator stopped working");
```

Comment:
Similar to the `doWork` method, the stop message should be more informative and specific. Change it to 'Excavator stopped working' to clearly indicate which machine has stopped.